packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -110,18 +110,21 @@ export default class RouterService extends Service {
      ```
 
      @method transitionTo
-     @param {String} routeNameOrUrl the name of the route or a URL
-     @param {...Object} models the model(s) or identifier(s) to be used while
+     @param {String} [routeNameOrUrl] the name of the route or a URL
+     @param {...Object} [models] the model(s) or identifier(s) to be used while
        transitioning to the route.
      @param {Object} [options] optional hash with a queryParams property
-       containing a mapping of query parameters
+       containing a mapping of query parameters. May be supplied as the only
+      parameter to trigger a query-parameter-only transition.
```

Comment:
Can you make an inline code example for this case in the docs above?

Reply:
Yes, great callout! Will push in just a minute!

Reply:
9808da9